packages/lesswrong/lib/collections/localgroups/views.ts

```diff
@@ -39,10 +50,10 @@ ensureIndex(Localgroups, { organizerIds: 1, inactive: 1 });
 
 Localgroups.addView("all", function (terms: LocalgroupsViewTerms) {
   return {
-    options: {sort: {createdAt: -1}}
+    options: {sort: {name: 1}}
   };
 });
-ensureIndex(Localgroups, { createdAt: -1 });
+ensureIndex(Localgroups, { inactive: 1, name: 1 });
```

Comment:
Should this still have inactive on the index?

Reply:
I wasn't actually sure about that - the `inactive` filter is in the default view and it is getting applied when we use this view as well, so I assumed it should be included in this index?

Reply:
Oh, I see. In posts we have `augmentForDefaultView`, but I don't think you need to do that here. This seems fine.